### ISSUE TITLE
[BUG] Fullscreen uses both monitors in error

### ISSUE THREAD

**Issue by vanessakindell** [IgeNiaI/Q-Zandronum#336]
Using Q-Zandronum 1.4.11 - 2023-11-26 18:04:04 -0500 - SDL version
(from the latest release here on GitHub.)

When I try to put the game in fullscreen, it takes both monitors as part of the screen instead of staying on the one.
I've tried tweaking numerous settings to try to resolve it and using both Software and OpenGL render options to no avail.

--In-game console outputs--
vid_displayadapters output: Multi monitor support unavailable
vid_adapter output: Unknown command "vid_adapter"

Full command line output https://pastebin.com/cmNtHDv1
My neofetch info: https://pastebin.com/nndsDTmV
My system info: https://pastebin.com/bgwPytGp

**Comment by IgeNiaI** [comment]
Does this happen on vanilla Zandronum too?

**Comment by Zardoz89** [comment]
Yes. Its more related to the fact that Zandroum and Q-Zandronum are using an ancient version of SDL. GZDoom have fixed this issues. And I try to improve this on Linux ( https://github.com/ZDoom/gzdoom/pull/1577 )

The Zandronum/Q-Zandronum should import the HUGE changes that GZDoom did on the video side of the code. I try to port this changes, but sadly I lack the know of the Zandroum and GzDoom code to do this.
